cytominer_eval/operations/percent_strong.py

```diff
@@ -38,13 +39,17 @@ def percent_strong(
     # Check to make sure that the melted dataframe is upper triangle
     assert_melt(similarity_melted_df, eval_metric="percent_strong")
 
+    # check that there ARE group_replicates (non-unique rows)
+    replicate_df = similarity_melted_df.query("group_replicate")
+    denom = replicate_df.shape[0]
+
+    ### HYT's addition, though i dont know if this will work
```

Comment:
I think it will! But, we do need to confirm before merging the PR

To confirm, please add a test to [`test_percent_strong.py`](https://github.com/cytomining/cytominer-eval/blob/master/cytominer_eval/tests/test_operations/test_percent_strong.py) that will fail this assert.

You can follow the pattern [here](https://github.com/cytomining/cytominer-eval/blob/master/cytominer_eval/tests/test_operations/test_util.py#L76-L81) to confirm the assertion.

Let's also get rid of the comment on line 46 🙃 

In general, I believe that code attribution is EXTREMELY (emphasis added here!!) important. Luckily, github has a nice "blame" feature to determine line-by-line attribution.